sdk/src/assertions/asset_reference.rs

```diff
@@ -0,0 +1,145 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    assertion::{Assertion, AssertionBase, AssertionCbor},
+    assertions::labels,
+    error::Result,
+};
+
+const ASSERTION_CREATION_VERSION: usize = 2;
```

Comment:
There was no previous version of this assertion, so we don't need to specify one here. We can use the default.